frontend/test/acceptance/pages/EnterDetails.scala

```diff
@@ -0,0 +1,80 @@
+package acceptance.pages
+
+import acceptance.Config.baseUrl
+import acceptance.{TestUser, Util}
+import org.openqa.selenium.{WebElement, By, WebDriver}
+import org.scalatest.selenium.{WebBrowser, Page}
+
+class EnterDetails(implicit val driver: WebDriver) extends Page with WebBrowser with Util {
+  val url = s"$baseUrl/join/partner/enter-details"
+
+  def userDisplayName: String = {
+    val selector = cssSelector(".js-user-displayname")
+    assert(pageHasElement(selector))
+    selector.element.text
+  }
+
+  def becomePartner = {
+    val selector = cssSelector("a[href='/join/partner/enter-details']")
+    assert(pageHasElement(selector))
+    click on selector
+  }
+
+  def pageHasLoaded(): Boolean = {
+    pageHasElement(id("cc-cvc"))
```

Comment:
It may be worth switching these to have a `qa-` prefix as per [these guidelines](https://github.com/guardian/membership-frontend/blob/master/docs/client-side-principles.md), once you've checked that nothing else is using the ids. This can go in a later iteration if necessary